### PR TITLE
Write "West" correctly in italic

### PR DESCRIPTION
**Pull request by miago**

### CODE REVIEW

File: _posts/2024-01-10-practical_zephyr_basics.md [hunk 1]
```diff
@@ -693,4 +693,4 @@ Finally, have a look at the files in the [accompanying GitHub repository](https:
 
 [zephyr-gh-docker]: https://github.com/zephyrproject-rtos/docker-image
 
--->
\ No newline at end of file
+-->
```
Comment (gminn) [inline _posts/2024-01-10-practical_zephyr_basics.md:696]: Could you revert this removal of the new line?

**Review by gminn — CHANGES_REQUESTED** [review]
@miago thank you for submitting this typo fix! I have one request before I approve the PR

File: _posts/2024-01-10-practical_zephyr_basics.md (hunk 1 above)
Comment (miago) [reply _posts/2024-01-10-practical_zephyr_basics.md]: yes, sorry!

Comment (gminn) [reply _posts/2024-01-10-practical_zephyr_basics.md]: No problem, thank you!

**Review by gminn — APPROVED** [review]

**Pull request merged**